### PR TITLE
strings are now valid matlab so use some other example

### LINKED ISSUE #241: update `+XFAIL_IF(MATLAB)` example
In `help doctest`, there is an example of matlab/octave conditional XFAIL.

I think `"string"` is now valid matlab, so we need a different example.

### PR DESCRIPTION
**Pull request by cbm755**
Fixes #241.

### CODE REVIEW

**Comment by cbm755** [comment]
Personally I find directly indexing into a function output one of the nicer extensions which I miss in Matlab.  But maybe its not very clear to non-expert users?

@catch22 to you like what I've done in this PR?  Or would you prefer:
```
upper({'GNU', 'Octave'}){2}
>> OCTAVE
```
Or maybe someone has a better example?

**Comment by mtmiller** [comment]
How about

```
>> license
ans = GNU General Public License
```

?

**Comment by cbm755** [comment]
Love this on so many levels!

**Comment by cbm755** [comment]
@catch22 ok to merge?

File: inst/doctest.m [hunk 1]
```diff
@@ -165,8 +165,8 @@
 %%
 %% @example
 %% @group
-%% >> "shiny Octave feature"    % doctest: +XFAIL_IF(DOCTEST_MATLAB)
-%% ans = shiny Octave feature
+%% >> license       % doctest: +XFAIL_IF(DOCTEST_MATLAB)
+%% ans = GNU General Public License
```
Comment (cbm755) [inline inst/doctest.m]: This is indeed the shiniest possible feature any software can have :-)

This must make @mtmiller a contender for greatest short commit ever!

**Pull request merged**